packages/eslint-config/configs/wordpress.js

```diff
@@ -0,0 +1,103 @@
+/**
+ * The original version of this file is based on ESLint shareable config for WordPress:
+ * https://github.com/WordPress-Coding-Standards/eslint-config-wordpress.
+ */
+
+module.exports = {
+	env: {
+		browser: true,
```

Comment:
I think this is a good opportunity to eliminate undesirable globals which have shown to be problematic in the past, as evidenced in https://github.com/WordPress/gutenberg/pull/12070#issuecomment-439963253, https://github.com/WordPress/gutenberg/pull/9163#discussion_r211256330, and https://github.com/WordPress/gutenberg/pull/1332#issuecomment-310053886

Reply:
Agreed, this has been troublesome in the past

Reply:
Cleaned up in d5256b4.